sample.audius.conf

```diff
@@ -1,13 +1,30 @@
 # creator-node audius.conf
 CADDY_TLS='tls internal'
 creatorNodeEndpoint=https://localhost
-delegateOwnerWallet=0xb3c66e682Bf9a85F6800c769AC5A05c18C3F331d
-delegatePrivateKey=62ca4962f99a39b4a2c60a8402f60baea5437c275146abd90e0bfdcf0415184d
+delegateOwnerWallet=0x0D38e653eC28bdea5A2296fD5940aaB2D0B8875c
+delegatePrivateKey=21118f9a6de181061a2abd549511105adb4877cf9026f271092e6813b7cf58ab
 spOwnerWallet=0xb3c66e682Bf9a85F6800c769AC5A05c18C3F331d
+web3EthProviderUrl=http://host.docker.internal:8546
 
 # discovery-provider audius.conf
 audius_discprov_url=https://localhost
 audius_delegate_owner_wallet=0xfF4329197C7bD52480567CE6c19331BD37555B3c
 audius_delegate_private_key=c1c36935d3809daacf3aa63243feef5e7a97ba11dc56fe3a923758937362ed6e
+audius_web3_eth_provider_url=http://host.docker.internal:8546
+audius_eth_contracts_registry='0xABbfF712977dB51f9f212B85e8A4904c818C2b63'
+audius_eth_contracts_token='0xdcB2fC9469808630DD0744b0adf97C0003fC29B2'
+audius_solana_endpoint=http://host.docker.internal:8899
 
-solanaClaimableTokenProgramAddress=testHKV1B56fbvop4w6f2cTGEub9dRQ2Euta5VmqdX9
+# identity-service audius.conf
+ethProviderUrl=http://host.docker.internal:8546
+ethTokenAddress='0xdcB2fC9469808630DD0744b0adf97C0003fC29B2'
+ethRegistryAddress='0xABbfF712977dB51f9f212B85e8A4904c818C2b63'
+ethOwnerWallet='0x855FA758c77D68a04990E992aA4dcdeF899F654A'
+solanaEndpoint=http://host.docker.internal:8899
+solanaClaimableTokenProgramAddress='testHKV1B56fbvop4w6f2cTGEub9dRQ2Euta5VmqdX9'
+
+audius_auto_upgrade_enabled='false'
+AUDIUS_DOCKER_COMPOSE_GIT_SHA='a431e2e425cc97af857c11c2f99d27cdd3910231'
+
+# audius-d conf
+ethereumProviderUrl=http://localhost:8546
```

Comment:
I'm open to eliminating unnecessary config here, but first we want to confirm chain registration is picked up by all the services.

Reply:
my vote is we address this in the config / flag refactor and temporarily inflate this file for now

Reply:
Just noting here as well that I've separated this config to register/ so that CI still works but we can still easily dev and iterate on the registration workflow using `audius-d -c register/sampleregister.audius.conf register`